app/views/admin/dashboard/show.html.erb

```diff
@@ -11,13 +11,11 @@
     <th>Actions</th>
   </tr>
   <% @facade.tutorials.each do |tutorial| %>
-    <tr class="admin-tutorial-card">
+    <tr class="admin-tutorial-card" id = "tutorial-<%= tutorial.id %>" >
       <td><%= link_to tutorial.title, tutorial_path(tutorial) %></td>
       <td>
-        <%= link_to "Edit", edit_admin_tutorial_path(tutorial) %> |
-        <%= link_to 'Destroy', tutorial_path(tutorial),
-                               method: :delete,
-                               data: { confirm: 'Are you sure?' } %>
+        <%= link_to "Edit", edit_admin_tutorial_path(tutorial) %> 
+        <%= button_to 'Destroy', "/admin/tutorials/#{tutorial.id}", method: :delete, data: { confirm: 'Are you sure?' }, class: "btn btn-primary mb1 bg-teal"%>
```

Comment:
Thanks for looking into this button/link issue. We can always come back and re-visit this later after check in on Monday.